.travis.yml

```diff
@@ -9,15 +10,17 @@ notifications:
 
 before_install:
   - chmod +x mvnw
-  - COMMITS_SINCE_MAIN=$(git rev-list --count HEAD)
-  - TAG_NAME="${TRAVIS_BRANCH}-m.${COMMITS_SINCE_MAIN}"
-  - VERSION_NAME="0.0.1-${TAG_NAME}"
-  - echo "Version ${VERSION_NAME}"
-  - ./mvnw versions:set -DnewVersion="${VERSION_NAME}"
+
 
 os: linux
 dist: bionic
 jdk: openjdk11
 script:
-  - ./mvnw test
-  - ./mvnw compile jib:build -Dversion="${VERSION_NAME}"
\ No newline at end of file
+  - ./mvnw test --no-transfer-progress
+
+deploy:
+  provider: script
+  script: bash deploy.sh
+  skip_cleanup: true
+  on:
+    branch: release
```

Comment:
After this PR is merged, I will create the "release" branch.
Deployments should only be triggered when there are commits in the "release" branch